samples/hello-world/HW-qio.ipynb

```diff
@@ -0,0 +1,265 @@
+        ")"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {}
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Let's use the `Workspace` object to view the available _targets_."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "targets = workspace.get_targets()\r\n",
+        "print(\"This workspace's targets:\")\r\n",
+        "for target in targets:\r\n",
+        "    print(\"-\", target.name)"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "source_hidden": false,
+          "outputs_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The resulting list should contain a list of target names, along with some details for each one. A _target_ is a remote solver that can perform optimization or quantum computation. In this example, we will focus on optimization."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### ❕ Do you see `microsoft.simulatedannealing-parameterfree.cpu` in your list of targets? If so, you're ready to keep going.\r\n",
+        "\r\n",
+        "Don't see it? You may need to add Microsoft QIO to your workspace to run this sample. Navigate to the **Providers** page in the portal and click **+Add** to add the Microsoft QIO provider."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Microsoft QIO: The optimization provider\r\n",
+        "Azure Quantum connects with first- and third-party optimizers to deliver optimization solutions. These offerings are called *providers*. Each provider can offer multiple *targets* with different capabilities. See the table below for Microsoft QIO's targets.\r\n",
+        "\r\n",
+        "**Microsoft QIO target ID anatomy**: `microsoft.{algorithm name}.{hardware type}`. Algorithm names ending in `-parameterfree` use automatic parameter selection. \r\n",
+        "\r\n",
+        "Optimization Algorithm | Target IDs | Python Class\r\n",
+        "| --- | ---| ---| --- | -- |\r\n",
+        "[Simulated annealing](https://docs.microsoft.com/azure/quantum/optimization-simulated-annealing) | `microsoft.simulatedannealing.cpu`,<br/>`microsoft.simulatedannealing-parameterfree.cpu`,<br/>`microsoft.simulatedannealing.fpga` |  `azure.quantum.optimization.SimulatedAnnealing` |\r\n",
```

Comment:
"`azure.quantum.optimization.SimulatedAnnealing`" should be replaced with "`microsoft.simulatedannealing-parameterfree.fpga`" (see: https://github.com/microsoft/qdk-python/blob/9e70ec7517830a36ef5615c40839d6c8272e28c4/azure-quantum/azure/quantum/target/microsoft/qio/simulated_annealing.py#L17)

Reply:
Ah that column is for the python class, the previous ones are for the targets.  It's a bit tricky to parse the tables because github does not render them

Reply:
But I'll add the FPGA simulated annealing!

Reply:
Oh gotcha, yes my mistake!